src/utils/legendmaker.js

```diff
@@ -109,15 +120,28 @@ export const renderLegendItem = function renderLegendItem(svgIcon, label = '') {
           </li>`;
 };
 
+export const renderExtendedLegendItem = function renderExtendedLegendItem(extendedLegendItem) {
```

Comment:
Could this function be merged with renderLegendItem instead (so that there is just one function)? They are quite similar.

Reply:
I was thinking you could set the lines that are changed to a variable instead, so that the variable would have the correct value when the function is called, or set it is a parameter perhaps.

Reply:
@markus659, @imantahriri, what's your conclusion?

Reply:
@tonnyandersson I think it is good as is.